static/test/javascripts/spec/common/weather.spec.js

```diff
@@ -14,10 +14,10 @@ define([
     Injector
     ) {
 
-    return  new Injector()
+    return new Injector()
         .store('common/utils/config')
         .require(['common/modules/weather', 'mocks'], function (sut, mocks) {
-            ddescribe('Weather component', function () {
+            describe('Weather component', function () {
```

Comment:
Ah I see, although it looks like it means "only run suites prepended with d", which excluded the rest of our tests.